core/services/subscription.go

```diff
@@ -4,42 +4,35 @@ import (
 	"context"
 	"fmt"
 	"math/big"
-	"sync"
 	"time"
 
-	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/jpillora/backoff"
+	"github.com/smartcontractkit/chainlink/core/gracefulpanic"
 
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/services/eth"
 	strpkg "github.com/smartcontractkit/chainlink/core/store"
 	"github.com/smartcontractkit/chainlink/core/store/models"
-	"github.com/smartcontractkit/chainlink/core/store/orm"
-	"github.com/smartcontractkit/chainlink/core/store/presenters"
 	"github.com/smartcontractkit/chainlink/core/utils"
 
-	ethereum "github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum"
 	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 )
 
-// Unsubscriber is the interface for all subscriptions, allowing one to unsubscribe.
-type Unsubscriber interface {
```

Comment:
I think this was a premature abstraction - its only used for the InitiatorSubscription and without it we can log details when unsubscribing